_component_design/choice-tiles.md

```diff
@@ -55,19 +79,19 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/choice-tiles/
 
 ### Composition
 
-Cards are composed of the following:
+A choice tile contains the following elements:
 
-- **Choice tile**: the container that houses the choice tile selection.
-- **Icon**: an iconographic representation of the selection contained within a choice tile.
-- **Heading**: the title of the selection contained within a choice tile.
-- **Description** *(optional)*: A description briefly explains the choice tile selection to a user. Capitalize the first word in sentences.
+- **Container (required)**: The container holds the icon, heading, and the description.
+- **Icon (optional)**: The icon visually represents the choice tile selection.
+- **Option label (required)**: The option label is the title of the selection contained in the choice tile.
+- **Description (optional)**: A description briefly explains the choice tile selection to a user.
```

Comment:
explains the option or the impact of selecting the option you can select with the choice tile.